packages/browser/src/plugins/in-app-plugin/events.ts

```diff
@@ -0,0 +1,33 @@
+export enum InAppEvents {
+    MessageOpened = 'in-app:message-opened',
+    MessageDismissed =  'in-app:message-dismissed',
+    MessageError = 'in-app:message-error',
+    MessageAction = 'in-app:message-action'
+}
+
+export const allEvents:string[] = [InAppEvents.MessageOpened, InAppEvents.MessageDismissed, InAppEvents.MessageError, InAppEvents.MessageAction];
```

Comment:
Would this work? (just to avoid repeating the keys)

```suggestion
export const allEvents = Object.keys(InAppEvents);
```